libraries/kmp-iap/library/src/androidMain/kotlin/io/github/hyochan/kmpiap/openiap/WebhookTransport.android.kt

```
@@ -0,0 +1,161 @@
+                        val event = parsed.event
+                        if (event != null) {
+                            emit(event)
+                            parsed.eventId?.let { resumeId = it }
+                        } else if (parsed.shouldAdvanceCursorOnDrop) {
+                            parsed.eventId?.let { resumeId = it }
+                        }
+                        continue
+                    }
+                    frameLines.append(line).append('\n')
+                }
+            } catch (error: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)